src/App.tsx

```diff
@@ -21,6 +22,15 @@ const App = () => {
             <Routes>
                 <Route path="/" element={<Layout />}>
                     {/* <Route index element={<HomePage />} /> */}
+                    <Route
+                        index
+                        element={
+                            <LandingPage
+                                parentButtonText="I'm a parent"
```

Comment:
Don't neet to pass this props, it is static data, it doesn't any effect.

Reply:
done